server/requirements.txt

```diff
@@ -57,15 +57,13 @@ click==7.1.2
     # via flask
 croniter==0.3.37
     # via superdesk-core
-cryptography==41.0.7
+cryptography==42.0.7
     # via
     #   authlib
     #   jwcrypto
 deepdiff==3.3.0
     # via superdesk-planning
-deprecated==1.2.14
-    # via jwcrypto
-draftjs-exporter[lxml]==2.1.7
+draftjs-exporter[lxml]==2.1.0
```

Comment:
Looks like this is downgraded, compared to what you already have in production. Just something to keep in mind (tried locally, and same downgrade occurs)

Reply:
Yes, I did notice that!